pkg/cli/mapping_validate.go

```diff
@@ -33,6 +33,8 @@ type MappingValidateCommand struct {
 	cli.BaseCommand
 
 	flagPath string
+
+	verbosity bool
```

Comment:
I think we should standardize on `log-level` rather than introducing another flag and leave it to the logger. See https://github.com/abcxyz/pkg/issues/121. We decided to punt this until we introduce `slog` package. Maybe we should punt this issue in pmap as well.